doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -91,68 +84,84 @@ and tables:
 Flags, options and tables are identified by a series of identifiers, each with an initial
 capital letter.
 
-.. cmd::  Set @flag
+.. cmd:: {? Export } Set {+ @ident }
```

Comment:
> the fact that flags, options and tables are identified by a series of space separated idents is an implementation detail that should not be exposed to the reader

Huh?  Isn't it completely obvious if you glance at the list of flags and options?  And the user does need to type these spaces.  I'll re-introduce `option_table`, which I spliced earlier, and rename it to `setting_name`.  And use that for `Add` and `Remove`.  That will simplify the descriptions and syntax.

> this fact is even uglier and thus more reasonable to hide when you discover that tables can be identified by at most two identifiers

Since AFAIK we don't have user-defined tables, users will never be aware of this.

Take a look what I have now.

Reply:
Thanks, your new version looks good to me. But in this case, you can remove the `:production:` from the bullet-point list above (and simply replace them with bold).

Reply:
I made it bold.  Elsewhere I've used italics when I'm introducing a defined term.  This seems like a defined term, so should it just be italic?

Reply:
> But it should not be displayed here, like `Local` and `Global` are not displayed either.

Leaving `Export` undocumented :-(

Reply:
Italic seems good as well.

`Export` *is* documented. Cf. https://coq.gitlab.io/-/coq/-/jobs/491537737/artifacts/_install_ci/share/doc/coq/sphinx/html/proof-engine/vernacular-commands.html#locality-attributes-supported-by-set-and-unset.

BTW, there is a discrepancy between various quoting strategies. Sometimes, you use `:n:` quoting just to get bold even if it doesn't contain any notation. This is a bit heavy. It would seem that single backticks behave the same, whereas double backticks give something that much less visible (and not bold). Finally, I don't know what we should do about `:g:` but currently people don't know how to use it and it brings no benefit.

Reply:
I'll try to use single backticks going forward.